application/templates/cms/measure_actions.html

```diff
@@ -3,18 +3,18 @@
         <div class="measure-action-section__main" id='measure-action-section-{{ measure.guid }}'>
             <div style="">
                 {% if measure.status == 'DRAFT' %}
-                    <a href="{{ url_for('cms.edit_measure_page', topic_uri=topic.uri, subtopic_uri=subtopic.uri, measure_uri=measure.uri, version=measure.version) }}" id="measure-action-section__edit_button-{{ measure.guid }}">Edit</a>
+                    <a href="{{ url_for('cms.edit_measure_page', topic_uri=topic.uri, subtopic_uri=subtopic.uri, measure_uri=measure.uri, version=measure.version) }}" id="measure-action-section__edit_button-{{ measure.guid }}">edit</a>
```

Comment:
I think the lowercasing of link text here looks a bit weird on the page, especially when the measure name and status are capitalised.

![screenshot from 2018-11-01 13-51-46](https://user-images.githubusercontent.com/6525554/47855812-75488d00-dddd-11e8-935e-6d1ab2c31af8.png)

Reply:
@TheDoubleK thanks for your feedback! That was a deliberate choice by @Nadiahuq but I can also see your point. I can't see anything in the [Style Guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style) about this.

Reply:
👍